### PR TITLE
Changing the readme text in the fork

### PR DESCRIPTION
**Pull request by ceej640**
^

### CODE REVIEW

**Comment by ceej640** [comment]
@mkitti this is my pull req

File: README.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ Here we are demonstrating a pull request.
 
 I'm making this change on the main branch.
 
-DB is adding to fork for funsies.
+CJ's fork changed the text
```
Comment (mkitti) [inline README.md:10]: This is a much better sentence.

File: README.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ Here we are demonstrating a pull request.
 
 I'm making this change on the main branch.
 
-DB is adding to fork for funsies.
```
Comment (mkitti) [inline README.md:10]: Funsies is not a word!

**Review by mkitti — APPROVED** [review]
Looks good!

**Pull request merged**